x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/hooks/use_non_closed_alerts.ts

```diff
@@ -35,6 +36,11 @@ export const useNonClosedAlerts = ({
     queryId,
     to,
     from,
+    runtimeMappings: {
+      [EntityIdentifierFields.related]: {
+        type: 'keyword',
+      },
+    },
```

Comment:
I've changed this to be a prop spread of passed runtimeMappings, lmk if there was a reason you implemented it that way on purpose